homeassistant/components/shopping_list/__init__.py

```diff
@@ -50,6 +52,13 @@
     {vol.Required("type"): WS_TYPE_SHOPPING_LIST_ADD_ITEM, vol.Required("name"): str}
 )
 
+SCHEMA_WEBSOCKET_REMOVE_ITEM = websocket_api.BASE_COMMAND_MESSAGE_SCHEMA.extend(
```

Comment:
Nowadays we put the command schema as a decorator to the command.

Example:
https://github.com/home-assistant/core/blob/80debae96d002eaa210fb0661280e20341a443c1/homeassistant/components/analytics/__init__.py#L40-L53

https://github.com/home-assistant/core/blob/80debae96d002eaa210fb0661280e20341a443c1/homeassistant/components/analytics/__init__.py#L33